### ISSUE TITLE
Selectively enable pinch-to-zoom

### ISSUE THREAD

**Issue by dae** [ankitects/anki#985]
Currently webview.py's eventFilter() captures native gesture events and ignores them to prevent pinch-to-zoom,  because:

- the top and bottom webviews, and places like the deck list and editing screen, do not make sense to zoom
- issues have been reported in the past where the reviewing screen behaved oddly after zooming in and then answering a card

If the latter issue could be fixed/confirmed to no longer exist, selectively enabling pinch during review might be worth looking into.

Of further note, the event filter doesn't appear to work reliably anymore in recent Qt versions. On a Mac here for example, pinching in the main screen starts off, but once the congratulations screen is shown, pinching is now possible, even when returning to the deck list.

Related: https://forums.ankiweb.net/t/feature-request-enable-pinch-to-zoom-on-trackpad/5333

**Comment by georgez-zgeorge** [comment]
Just a note on this. This is my current set up:
```
    Anki 2.1.38 (355e4cd5) Python 3.8.6 Qt 5.14.2 PyQt 5.14.2
    Platform: Mac 10.16
    Flags: frz=True ao=True sv=2
    Add-ons, last update check: 2021-02-17 15:38:56
```
I'm unable to pinch to zoom at all until I see an congratulations screen. I can't tell you how long I've been working to figure out what combination of settings, addons, closing my computer, sitting facing north, and other crazy configurations I tried to figure out why pinch to zoom only worked _sometimes_ until I found this post. Thank you for figuring out this bug and I look forward to being able to zoom at any time on the review screen.
